modules/labs-react/pagination/lib/Pagination/PageButton.tsx

```diff
@@ -17,14 +17,15 @@ const StyledPageButton = styled(IconButton)<{toggled?: boolean}>(
     ...type.small,
   },
   ({toggled}) => {
-    return toggled ? toggledStlyes : {};
+    return toggled ? toggledStyles : {};
   }
 );
 
-export interface PageButtonProps extends IconButtonProps {
-  model: PaginationModel;
-  pageNumber: number;
-}
+export type PageButtonProps = IconButtonProps &
+  React.ButtonHTMLAttributes<HTMLButtonElement> & {
```

Comment:
When `PageButton` uses `createComponent`, the props will no longer need to extend from `React.ButtonHTMLAttributes`